apps/utils/locust_tests/common/validation.py

```diff
@@ -21,8 +21,8 @@ class ValidationGoal(Enum):
     SLA_PATIENT = (1000, 3000, 5000)
     SLA_EOB_WITH_SINCE = (100, 250, 1000)
     SLA_EOB_WITHOUT_SINCE = (500, 1000, 3000)
-    SLA_V1_BASELINE = (500, 700, 1000)
-    SLA_V2_BASELINE = (500, 700, 1000)
+    SLA_V1_BASELINE = (700, 1000, 3000)
+    SLA_V2_BASELINE = (700, 1000, 3000)
```

Comment:
Not in this PR, but would it make sense to get these values from a configuration or SSM or something? I can think of two scenarios where that could be helpful — having a single source of truth for multiple uses and making them easy to change — but I'm not sure either of those is applicable here.